sdk/textanalytics/ai-text-analytics/samples-dev/alternativeDocumentInput.ts

```diff
@@ -2,15 +2,14 @@
 // Licensed under the MIT License.
 
 /**
- * Demonstrates how to use object inputs as alternatives to
- * string arrays in client methods
+ * @summary uses object document inputs with attached metadata rather than simple strings for more flexibility
  */
 
 import {
   TextAnalyticsClient,
   AzureKeyCredential,
   DetectLanguageInput,
-  TextDocumentInput,
+  TextDocumentInput
```

Comment:
seems like sometimes folks prettier config is adding the trailing comma but not always. I'm not sure why this is happening tbh

Reply:
This reminds me to include in the migration guide a step to add `samples-dev` to the formatting commands. This happens because sample code was previously excluded from `rushx format`.